src/transform.ts

```diff
@@ -133,22 +144,26 @@ export interface WindowTransform {
   window: WindowFieldDef[];
 
   /**
-   * A two element specification about how large the sliding window is. The first element indicates the start and the second element indicates the end. If `null` is specified for the start, it will include everything before the current point. If `null` is specified for the end, it will include everything after the endpoint. For example a frame of `[-5,5]` says the window should include 5 previous objects and 5 after objects. The default is `[null, 0]`, which means include everything in the window. `[null, null]` would mean include everything in the window.
+   * A frame specification as a two-element array indicating how the sliding window should proceed. The array entries should either be a number indicating the offset from the current data object, or null to indicate unbounded rows preceding or following the current data object. The default value is `[null, 0]`, indicating that the sliding window includes the current object and all preceding objects. The value `[-5, 5]` indicates that the window should include five objects preceding and five objects following the current object. Finally, `[null, null]` indicates that the window frame should always include all data objects.
+   *
+   * __Default value:__:  `[null, 0]` (includes the current object and all preceding objects)
    */
   frame?: (null | number)[];
 
   /**
-   * Will indicate whether to ignore peer values (items with the same rank) in the window. The default value is `False`.
+   * Will indicate whether to ignore peer values (items with the same rank) in the window.
+   *
+   * __Default value:__ `false'
    */
   ignorePeers?: boolean;
 
   /**
-   * The names of the data fields to partioin the objects into seprate windows. If not specified, everything will be in a single group.
+   * The data fields for partitioning the data objects into separate windows. If unspecified, all data points will be a single group.
    */
   groupby?: string[];
 
   /**
-   * A definition for sorting the objects within the window. Equivalent objects are considered a peer (Look at ignorePeers). If left undefined, the order of items in the window is undefined.
+   * A definition for sorting the data objects within the window. Equivalent objects are considered a peer (Look at ignorePeers). If undefined, the order of items in the window is undefined.
```

Comment:
Why "Look at ignorePeers" here?  (Looking at ignorePeers doesn't seem to make me understand `sort` better.)

Also, the description of this field in Vega docs is more descriptive.